source/components/temp_doeclim_component.cpp

```diff
@@ -248,7 +250,7 @@ void TempDOECLIMComponent::prepareToRun() throw ( h_exception ) {
     //double KTB2[ns];
     //double KTA3[ns];
     //double KTB3[ns];
-    
+
```

Comment:
All these lines of commented-out code need to be dealt with.  If we don't need them, then delete them.  If we do, then enable them (and don't make them fixed-size arrays).

Also, on line 243 we define our own value of pi.  Don't do that.  Use the ones in the standard math libraries.